libs/core/models/pagination.dto.ts

```diff
@@ -0,0 +1,44 @@
+
+/** T type information of each fetch. */
```

Comment:
Your models shouldnt know anything about fetching.
```suggestion
/** Pagination. */
```